tutorials/measurement_chain.ipynb

```diff
@@ -0,0 +1,553 @@
+{
```

Comment:
<p><code>GenericEquipment -&gt; MeasurementEquipment</code></p><p> </p><p><code><span class="ql-cursor">﻿﻿</span></code><span class="markdownDiffSpan" style="background: rgb(209, 255, 221) none repeat scroll 0% 0%; display: inline-block; --darkreader-inline-bgcolor:#114627; --darkreader-inline-bgimage:none;" data-darkreader-inline-bgcolor="" data-darkreader-inline-bgimage="">define all your instruments and once and reuse their defs -&gt; define all your instruments once, and reuse their defs.</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BAMWelDX/weldx/pull/326/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>